### PR TITLE
fix return impacts when active power control modification

### PR DESCRIPTION
**Pull request by achour94**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java [hunk 1]
```diff
@@ -765,37 +765,47 @@ public Reporter modifyActivePowerControlAttributes(ActivePowerControl<?> activeP
...
+            if (droopInfo != null) {
+                activePowerControlAdder
+                        .withDroop(droop);
+                reports.add(ModificationUtils.getInstance().buildModificationReport(Double.NaN,
+                        droop,
+                        "Droop"));
+            } else {
+                activePowerControlAdder
+                        .withDroop(Double.NaN);
+            }
```
Comment (AbdelHedhili) [inline src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java:805]: isn't this slightly better ? 
```
activePowerControlAdder.withDroop(droop);
if (droopInfo != null) {            
    reports.add(ModificationUtils.getInstance().buildModificationReport(Double.NaN, droop, "Droop"));
}
```

File: src/test/java/org/gridsuite/modification/server/modifications/BatteryModificationTest.java [hunk 2]
```diff
@@ -217,6 +218,40 @@ public void testDroopUnchanged() throws Exception {
...
+        modificationToCreateJson = mapper.writeValueAsString(batteryModificationInfos);
+        mvcResult = mockMvc.perform(post(getNetworkModificationUri()).content(modificationToCreateJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk()).andReturn();
+        //check impacts
+        resultAsString = mvcResult.getResponse().getContentAsString();
+        networkModificationResult = mapper.readValue(resultAsString, NetworkModificationResult.class);
+        assertEquals(1, networkModificationResult.getNetworkImpacts().size());
+        assertEquals(1, networkModificationResult.getImpactedSubstationsIds().size());
+        assertEquals("[s2]", networkModificationResult.getImpactedSubstationsIds().toString());
+
```
Comment (Meklo) [inline src/test/java/org/gridsuite/modification/server/modifications/BatteryModificationTest.java:252]: Wouldn't it be interesting to also check the battery ActivePowerControl values after the modifications ?

Comment (achour94) [reply src/test/java/org/gridsuite/modification/server/modifications/BatteryModificationTest.java:252]: this is not the purpose of this PR, the test of modification is done in the other methods

**Review by AbdelHedhili — APPROVED** [review]

File: src/test/java/org/gridsuite/modification/server/modifications/BatteryModificationTest.java (hunk 2 above)
Comment (Meklo) [reply src/test/java/org/gridsuite/modification/server/modifications/BatteryModificationTest.java:252]: Fair enough

**Pull request merged**